packages/client/mutations/SetDefaultSlackChannelMutation.ts

```diff
@@ -0,0 +1,77 @@
+import graphql from 'babel-plugin-relay/macro'
+import {commitMutation} from 'react-relay'
+import toTeamMemberId from '~/utils/relay/toTeamMemberId'
+import {StandardMutation} from '../types/relayMutations'
+import {SetDefaultSlackChannelMutation as TSetDefaultSlackChannelMutation} from '../__generated__/SetDefaultSlackChannelMutation.graphql'
+
+graphql`
+  fragment SetDefaultSlackChannelMutation_team on SetDefaultSlackChannelSuccess {
+    teamMember {
+      integrations {
+        slack {
+          notifications {
+            channelId
+          }
+        }
+      }
+    }
+  }
+`
+
+const mutation = graphql`
+  mutation SetDefaultSlackChannelMutation($slackChannelId: ID!, $teamId: ID!) {
+    setDefaultSlackChannel(slackChannelId: $slackChannelId, teamId: $teamId) {
+      ... on ErrorPayload {
+        error {
+          message
+        }
+      }
+      ...SetDefaultSlackChannelMutation_team @relay(mask: false)
+    }
+  }
+`
+export const setDefaultSlackChannelUpdater = (payload) => {
```

Comment:
nit: I like the suffix this with the subscription name + Updater. (e.g. setDefaultChannelSlackChannelTeamUpdater). it's verbose, but useful because on occasion, a mutation could push to 2 different subscription channels & each could have their own updater. (e.g. TeamUpdater, MeetingUpdater).
It's also a healthy sanity check because when you look in the TeamSubscription.ts file, you should be guaranteed to see that all updaters end in `TeamUpdater` 
(that said, i think it's probably best to not have a subscription, or change this to a notification subscription)